prime-angular-frontend/src/app/modules/adjudication/pages/health-authority-authorized-users-page/health-authority-authorized-users-page.component.ts

```diff
@@ -0,0 +1,30 @@
+import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { RouteUtils } from '@lib/utils/route-utils.class';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-health-authority-authorized-users-page',
+  templateUrl: './health-authority-authorized-users-page.component.html',
+  styleUrls: ['./health-authority-authorized-users-page.component.scss']
+})
+export class HealthAuthorityAuthorizedUsersPageComponent implements OnInit {
+  public busy: Subscription;
+
+  private routeUtils: RouteUtils;
+
+  constructor(
+    protected route: ActivatedRoute,
```

Comment:
Should be `private` only need to use `protected` for DI when passing to a base class through `super(...)`